audit/projects/k8s-staging-prometheus-adapter/buckets/artifacts.k8s-staging-prometheus-adapter.appspot.com/iam.json

```diff
@@ -0,0 +1,37 @@
+{
+  "bindings": [
+    {
+      "members": [
+        "group:k8s-infra-artifact-admins@kubernetes.io",
+        "projectEditor:k8s-staging-prometheus-adapter",
+        "projectOwner:k8s-staging-prometheus-adapter"
+      ],
+      "role": "roles/storage.legacyBucketOwner"
+    },
```

Comment:
Expected, all of the k8s-staging-prometheus-adapter files are from https://github.com/kubernetes/k8s.io/pull/2274